components/RatingsTable/index.tsx

```diff
@@ -1,3 +1,4 @@
+import Link from 'next/link';
```

Comment:
I think we have a wrapper for linking called `LinkWithSeason`. we should be using that here

Reply:
i disagree. We have no need to pass the season into these links.

Reply:
change made